nautobot/extras/forms.py

```diff
@@ -947,6 +949,47 @@ class WebhookFilterForm(BootstrapMixin, forms.Form):
     enabled = forms.NullBooleanField(required=False, widget=StaticSelect2(choices=BOOLEAN_WITH_BLANK_CHOICES))
 
 
+#
+# Secrets
+#
+
+
+class SecretForm(BootstrapMixin, CustomFieldModelForm, RelationshipModelForm):
+    """Create/update form for `Secret` objects."""
```

Comment:
Comment for me: This seems to be the only page where we use Markdown ticks? I see that there are other forms in this file that do the same thing, but it doesn’t seem to be universal. Which way if preferred?

Reply:
Fair question. I don't think we have an official position on this yet. In general I'd like to see a lot of improvement to our docstrings (many of the forms in this file don't even have docstrings at all) and that's maybe something we should revisit then.